src/Core/src/Hosting/MauiAppBuilder.cs

```diff
@@ -56,6 +57,8 @@ internal MauiAppBuilder(bool useDefaults = true)
 				this.ConfigureCrossPlatformLifecycleEvents();
 				this.ConfigureDispatching();
 			}
+
+			this.UseEssentials();
```

Comment:
Should this go inside `if (useDefaults)`? Just an open question, I have no horses and this isn't a horse race...

Reply:
I had this thought too... Maybe not? but I believe we depend on some things happening here like on android for permissions that some API's will not work if they aren't wired up to the right lifecycle events...

I'm could go either way on this one...

Reply:
Yeah I don't have background on this unfortunately. Another way to look at it, is it "more" or "less" needed than what's already in the `if()`? Or about the same?

Reply:
Yea I think I'll move this inside, given that things like Handlers and Dispatching are in there already, this feels about the same level of 'importantness'